### PR TITLE
Temporarily exclude PatchModule*Test_PlatformModPatchModule on AIX

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
Temporarily exclude PatchModule*Test_PlatformModPatchModule on AIX
Signed-off-by: Mesbah Alam <Mesbah_Alam@ca.ibm.com>

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
Note: this PR includes exclusion of JlinkTest_RequiredMod test from osx due to https://github.com/AdoptOpenJDK/openjdk-systemtest/issues/203 as well.

File: systemtest/playlist.xml [hunk 1]
```diff
@@ -2161,6 +2163,7 @@
 		<groups>
 			<group>system</group>
 		</groups>
+		<platformRequirements>^arch.ppc</platformRequirements>
```
Comment (sophia-guo) [inline systemtest/playlist.xml]: ppcle will also be excluded. Is that expected?

Comment (Mesbah-Alam) [reply systemtest/playlist.xml]: yes, need to exclude from all AIX builds.

Comment (smlambert) [reply systemtest/playlist.xml]: os.aix ?

Comment (Mesbah-Alam) [reply systemtest/playlist.xml]: hmm.. that's better, changing to os.aix

**Pull request merged**